src/travel-time.js

```diff
@@ -33,9 +33,22 @@ export default class TravelTimeEntity {
 
   get time() {
     let time = this._entity.state || 0;
+    let timeHourMinutesRe = new RegExp('^(\\d+) hour (\\d+) mins$', 'ig');
```

Comment:
Technically, we could also just used a single regular expression for both scenarios, but I wanted to keep the code simple and avoid larger changes.